pkgs/development/python-modules/apycula/default.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, setuptools-scm
+, numpy
+, pandas
+, pillow
+, crcmod
+, openpyxl
+}:
+
+buildPythonPackage rec {
+  pname = "Apycula";
```

```suggestion
  pname = "apycula";
```

see https://nixos.org/manual/nixpkgs/stable/#contributing

---

Fixed.  Thank you for the review!